src/factory/load_gltf.rs

```diff
@@ -319,6 +403,6 @@ impl super::Factory {
             meshes.insert(i, instances);
         }
 
-        (group, cameras, meshes)
+        (group, cameras, meshes, clips)
```

Comment:
Shall we return a `Gltf` struct instead?

```rust
struct Gltf {
    cameras: Vec<Camera>,
    clips: Vec<Clip>,
    meshes: Vec<Mesh>,
    scenes: Vec<Group>,
}
```

Reply:
I think it's better, but not the necessity